docs/codeql/reusables/data-extensions.rst

```diff
@@ -0,0 +1,10 @@
+.. _data-extensions:
+
+Data Extensions
+===============
+
+.. pull-quote::
+
+   Beta Notice - Internal documentation
+
+   Data extensions are documented internally in a Google doc.
```

Comment:
Can you link to the Google doc?

Reply:
I will see if there is something we can link to - otherwise we will have to leave it as it is.

Reply:
In hindsight, maybe it's worth taking out the point about being in a Google doc and mentioning that it's available on request? I'm just thinking because technically this will be accessible by anyone, even if it's delisted from the TOC, right? I'll leave up to you either way 👍